core-bundle/contao/library/Contao/Image.php

```diff
@@ -73,6 +73,27 @@ public static function getPath($src)
 		return 'system/themes/' . $theme . '/images/' . $src;
 	}
 
+	/**
+	 * Get the URL to an image
+	 *
+	 * @param string $src The image name or path
+	 *
+	 * @return string The path-absolute or absolute URL depending on the assets context
+	 */
+	public static function getUrl($src)
+	{
+		$src = System::urlEncode(self::getPath($src));
+
+		if (!$src)
+		{
+			return '';
+		}
+
+		$context = str_starts_with($src, 'assets/') ? 'assets_context' : 'files_context';
```

Comment:
This should be inverted (to check for files) because we use the asset context e.g. for system/themes as well.

Reply:
But this check was always made this way: https://github.com/contao/contao/blob/2f5d8fffe300fa930942a3bff69c1ab57c473d29/core-bundle/contao/library/Contao/Image.php#L128